src/legacy/core_plugins/vis_type_metric/public/components/metric_vis_value.js

```diff
@@ -42,11 +42,14 @@ class MetricVisValue extends Component {
       'mtrVis__container-isfilterable': hasFilter,
     });
 
+    const handleClick = hasFilter ? this.onClick : null;
+
     const metricComponent = (
       <div
         className={containerClassName}
         style={{ backgroundColor: metric.bgColor }}
-        onClick={hasFilter ? this.onClick : null}
+        onClick={handleClick}
+        onKeyPress={handleClick}
```

Comment:
Nit: This handleClick function is available when a bucket aggregation is used. It's used to add a Filter. By assigning `handleClick` to `onKeyPress`, the Filter is added on any key press (but not for arrow keys, alt, ctrl). I think it make sense to limit it to the `Return` key.